operator/builtin/parser/csv/csv_test.go

```diff
@@ -197,14 +525,37 @@ func TestParserCSV(t *testing.T) {
 				p.FieldDelimiter = ","
 				p.LazyQuotes = true
 			},
-			`stanza "log parser",1,6ft,5`,
-			map[string]interface{}{
-				"name":   `stanza "log parser"`,
-				"age":    "1",
-				"height": "6ft",
-				"number": "5",
+			[]entry.Entry{
+				{
+					Body: "stanza \"log parser\",1,6ft,5",
+				},
+			},
+			[]interface{}{
+				map[string]interface{}{
+					"name":   "stanza \"log parser\"",
+					"age":    "1",
+					"height": "6ft",
+					"number": "5",
+				},
 			},
+			false,
+			false,
 		},
+		// {
+		// 	"parse-with-lazy-quotes",
+		// 	func(p *CSVParserConfig) {
+		// 		p.Header = "name,age,height,number"
+		// 		p.FieldDelimiter = ","
+		// 		p.LazyQuotes = true
+		// 	},
+		// 	`stanza "log parser",1,6ft,5`,
+		// 	map[string]interface{}{
+		// 		"name":   `stanza "log parser"`,
+		// 		"age":    "1",
+		// 		"height": "6ft",
+		// 		"number": "5",
+		// 	},
+		// },
```

Comment:
We can't have any commented out code. Does this need to be fixed or removed?

Reply:
Yeah that was a mistake forgot it was in there. Side effect of the rebase.